TelepresenceBot/mobile/src/main/res/layout/activity_human.xml

```diff
@@ -1,24 +1,27 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
   xmlns:tools="http://schemas.android.com/tools"
   android:layout_width="match_parent"
   android:layout_height="match_parent"
   android:orientation="vertical"
   tools:context="com.novoda.tpbot.human.HumanActivity">
 
-  <com.novoda.tpbot.SelfDestructingMessageView
+  <com.novoda.tpbot.support.SelfDestructingMessageView
     android:id="@+id/bot_controller_debug_view"
     style="@style/Controller.DebugText"
     android:layout_width="match_parent"
     android:layout_height="0dp"
     android:layout_weight="50"
     tools:text="This is some text yo" />
 
-  <com.novoda.tpbot.human.ControllerView
-    android:id="@+id/bot_controller_direction_view"
+  <com.novoda.tpbot.support.SwitchableView
```

Comment:
I didn't say that I prefer to see it done another way 😄 It was just a question.
I agree that `ViewSwitcher` is kinda weird to use. It is much straightforward with your custom view.